data-raw/ni-beds.R

```diff
@@ -0,0 +1,44 @@
+# Northern Ireland on Health Inequalities Explorer
+# Secondary care indicators - bed availability
+
+# ---- Load libs ----
+library(tidyverse)
+library(devtools)
+library(lubridate)
+
+# ---- Load internal sysdata.rda file with URL's ----
+load_all(".")
+
+query_url <-
+  query_urls |>
+  filter(id == "ni_beds") |>
+  pull(query)
+
+# ---- Download and wrangle data ----
+raw <- read_csv(query_url, skip=2)
+
+# Change dataset to tidy data with all relevant information
+# Removed "Day Case" column since it included missing values and duplicate values with "Total Day Cases".
+# Can calculate number of discharge beds = total available beds - total occupied beds
+ni_beds <- raw |>
+  select(
+    financial_year = "Financial Year",
+    quarter_ending = "Quarter Ending",
```

Comment:
Please can we encode this variable as class `date` and not `character`. The function `lubridate::as_date()` might come in handy here.

Reply:
You need to pass in a format:

```r
ni_beds |> 
  mutate(quarter_ending = as_date(quarter_ending, format = "%d/%m/%Y"))
```

Reply:
Please can we rename `quarter_ending` to `date`, which is a more common name used throughout the humaniverse?